ocd-2944.sql

```diff
@@ -0,0 +1,41 @@
+-- Update inheritance_errors_report table
+alter table openchpl.inheritance_errors_report
+add column if not exists certification_body_id integer;
```

Comment:
Should this big "bigint" instead of "integer" to match what's in the table definition in the other file? Also need to add a NOT NULL constraint at some point later in this script. Same comment with the other table.